docsrc/contributors/fx_converters.rst

```diff
@@ -0,0 +1,211 @@
+.. _dynamo_conversion:
+
+Dynamo Converters
+==================
+The dynamo converter library in Torch-TensorRT is located in ``TensorRT/py/torch_tensorrt/dynamo/conversion``.
+
+
+
+Steps
+==================
+
+Operation Set
+-------------------
+The converters in dynamo are produced by ``aten_trace`` and falls under ``aten_ops_converters`` ( FX earlier had ``acc_ops_converters``, ``aten_ops_converters`` or ``nn_ops_converters`` depending on the trace through which it was produced).  The converters are registered using  ``dynamo_tensorrt_converter`` for dynamo. The function decorated
+has the arguments - ``network, target, args, kwargs, name``,  which is common across all the operators schema.
+These functions are mapped in the ``aten`` converter registry dictionary (at present a compilation of FX and dynamo converters, FX will be deprecated soon), with key as the function target name.
+    
+    * aten_trace is produced by ``torch_tensorrt.dynamo.backend.compile``. The second round of trace is produced by ``aot_torch_tensorrt_aten_backend`` by invoking ``aot_module_simplified`` from ``torch._functorch.aot_autograd``
```

Comment:
Could be reworded to imply that `export` and `compile` trace slightly differently but both result in the same key set of ops.

> The traced ATen graph is produced by either `torch_tensorrt.dynamo.trace` (in the export path), or `aot_module_simplified` (in the compile path). Both of these tracers reduce the large set of Torch operators to the more concise ATen set.